README.md

```diff
@@ -177,6 +177,21 @@ const out = await llamaEndpoint.chatCompletion({
  max_tokens: 512,
 });
 console.log(out.choices[0].message);
+
+/// Using a third-party provider: 
+await inference.chatCompletion({
+ model: "meta-llama/Llama-3.1-8B-Instruct",
+ messages: [{ role: "user", content: "Hello, nice to meet you!" }],
+ max_tokens: 512,
+ provider: "sambanova"
+})
+
+await inference.textToImage({
+ model: "black-forest-labs/FLUX.1-dev",
+ inputs: "a picture of a green bird",
+ provider: "together"
+})
```

Comment:
I feel like the other provider example should be moved :arrow_up: , before the dedicated endpoints maybe? idk

Or the serverless examples should be moved done.

Anyway up to you & @gary149 @julien-c

Reply:
good point! thx